File: Reverse.java
import java.util.Arrays;

public class ReverseArray {
    
    public static int[] reverseArray(int[] arr) {
        int n = arr.length;
        int[] reversedArr = new int[n];
        int j = n - 1;
        for (int i = 0; i < n; i++) {
            reversedArr[j] = arr[i];
            j--;
        }
        return reversedArr;
    }

    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4, 5};
        int[] reversedArr = reverseArray(arr);
        System.out.println("Original Array: " + Arrays.toString(arr));
        System.out.println("Reversed Array: " + Arrays.toString(reversedArr));
    }
}

File: Reverse.py
def reverse_array(arr):
    n = len(arr)
    reversed_arr = [0] * n
    j = n - 1
    for i in range(n):
        reversed_arr[j] = arr[i]
        j -= 1
    return reversed_arr

arr = [1, 2, 3, 4, 5]
reversed_arr = reverse_array(arr)
print("Original Array:", arr)
print("Reversed Array:", reversed_arr)
